Berksfile

```diff
@@ -1,3 +1,9 @@
 source 'https://supermarket.chef.io'
 
+cookbook 'build-essential'
```

Comment:
Cookbooks that are both in the supermarket and in `metadata.rb` as dependencies don't need to be listed here.